### PR TITLE
chore(docs): add link to the typescript docs in readme

### PR DESCRIPTION
**Pull request by kkiani**
## Description

I want to include a link to the  typescript documentation file in the `README.md` for the project.

## Motivation and Context

The existing README fails to offer instructions on how to utilize the package effectively. As the README is shared across all released packages, it would be more practical to refer to the documentation in a separate file by providing a link in the README.

### CODE REVIEW

**Comment by shahinism** [comment]
@kkiani can we have an example stack folder? One practice I found useful with AWS products, is having a simple product example.

**Comment by kkiani** [comment]
> @kkiani can we have an example stack folder? One practice I found useful with AWS products, is having a simple product example.

add it under `sample` folder

File: README.md [hunk 1]
```diff
@@ -4,6 +4,12 @@ Cost Monitoring Construct is a CDK library that helps monitor costs for AWS clou
 
 With the Cost Monitoring Construct, you can share the responsibility of cost management between developers and business holders. This is achieved through the creation of meaningful reports that enable the business team to make informed decisions. Additionally, the Construct generates boilerplate code that can be used to apply these decisions in practice, making it easier to stay on top of your budget.
 
+# How to use Cost Monitoring Construct?
+
+To use Cost Monitoring Construct, all you need is to have the AWS CDK installed and set up. Once you have that, you can install the package from the repository of your choice.
+
+For more information on using this Construct in TypeScript checkout the [TypeScript documentation](docs/typescript.md).
```
Comment (shahinism) [inline README.md:11]: Now please also link to samples here.

Comment (kkiani) [reply README.md]: This is done @shahinism. Would you please approve it if you thinks is good for now?

**Review by shahinism — APPROVED** [review]

**Pull request merged**